GetBlogFeeds.cs

```diff
@@ -144,11 +144,11 @@ protected override async Task<IEnumerable<IDocument>> ExecuteInputAsync(IDocumen
               var speaker = input.GetString("Title");
               if (!string.IsNullOrWhiteSpace(speaker))
               {
-                context.LogWarning($"No RSS specified for {speaker}.");
+                context.LogInformation($"No RSS specified for {speaker}.");
```

Comment:
Changed these to information messages so they don't show up as PR checks since it's okay a speaker doesn't have a feed, we just want to know about it